package/windows/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG SERVERCORE_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${SERVERCORE_VERSION}
+
+RUN powershell -Command $ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue'; \
+    powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
+
+ADD Gemfile 'C:\fluentd\install\Gemfile'
+ADD Gemfile.lock 'C:\fluentd\install\Gemfile.lock'
+
+RUN choco install -y ruby --version 2.4.2.2 --params "'/InstallDir:C:\'" \
+&& choco install -y msys2 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby24\msys64'" \
+&& refreshenv \
+&& ridk install 2 3 \
+&& echo gem: --no-document >> C:\ProgramData\gemrc \
+&& gem install --file 'C:\fluentd\install\Gemfile' \
+&& gem sources --clear-all
+
+RUN powershell -Command $ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue'; \
```

Comment:
Is there any chance we can merge all the `RUN`? So that we can reduce the layer of the fluentd image.

Reply:
The dockerfile is reference from fluentd's dockerfile, in order to sync with fluentd, I prefer we do like they do. https://github.com/fluent/fluentd-docker-image/blob/master/v1.6/windows/Dockerfile